cmd/helm/package.go

```diff
@@ -109,6 +111,21 @@ func (p *packageCmd) run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if p.dependencyUpdate {
```

Comment:
I'm wondering if it might be better to create a new `downloader.Manager` here instead of calling the Cobra command.

My main reason for concern is just that sometime down the line, if somebody changes the dependencyUpdateCommand.run() method, they might not think of checking here to make sure things work.